oidc-controller/api/routers/oidc.py

```diff
@@ -113,6 +113,9 @@ async def get_authorize(request: Request, db: Database = Depends(get_db)):
 
     # Prepeare the presentation request
     use_public_did = not settings.USE_OOB_LOCAL_DID_SERVICE
+    wallet_did = client.get_wallet_did(public=use_public_did)
+
+    byo_attachment = PresentProofv10Attachment.build(pres_exch_dict["pres_request"])
```

Comment:
These were removed here
https://github.com/bcgov/vc-authn-oidc/pull/576/files

Not sure if merge conflict put them back or something